backend/api/Services/DeckService.cs

```diff
@@ -127,14 +133,25 @@ public async Task<Deck> Update(Deck deck)
             }
 
             context.Decks.Remove(deck);
-            await context.SaveChangesAsync();
+            await ApplyDatabaseUpdate(deck.Installation);
```

Comment:
Do we need to do the check? Only admin can access this endpoint.

Reply:
Yes, since if we change the permissions in the endpoint at some point in the future, it is unlikely they will remember to make the change here as well. And this does no harm in my opinion, it is best to not delegate security just to the controllers.